database/sql/V054__store_order_events.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TYPE OrderEventLabel AS ENUM (
+  'created',
+  'ready',
+  'filtered',
+  'invalid',
+  'executing',
+  'considered',
+  'traded',
+  'cancelled'
+);
+
+CREATE TABLE order_events (
+    order_uid bytea NOT NULL,
+    timestamp timestamptz NOT NULL,
+    label OrderEventLabel NOT NULL
+);
```

Comment:
In this PR you only define event insertion but we likely already know in what way we want to read them. For this there should be an index to make it efficient. You know this better than I but I would guess we want a sorted index (b-tree) over `(order_uid, timestamp)` to efficiently retrieve all the events for an order ordered by time.